node/node.go

```diff
@@ -870,6 +901,21 @@ func (n *Node) GetAccount(address string, options api.AccountQueryOptions) (api.
 	}, blockInfo, nil
 }
 
+func adaptBlockInfoIfError(err error) (api.BlockInfo, error) {
```

Comment:
The name is a bit imprecise (since the "thing to adapt" isn't received as a parameter). Could be `extendErrorWithBlockInfo` (just an option, perhaps not a very good one).

Could also be `detectEmptyAccount() -> (blockInfo, bool)` (as far as I see, the returned error type is never used - the error is simply compared to nil).

Reply:
renamed to `extractBlockInfoIfNewAccount`